.github/workflows/node-test.yaml

```diff
@@ -65,6 +65,18 @@ on:
         required: false
         type: string
 
+      action-repository:
+        description: "Repository to load the actions from"
+        default: ${{ github.repository || 'pkgs/action' }}
+        required: false
+        type: string
+
+      action-ref:
+        description: "Git ref to load the actions from"
+        default: ${{ github.ref || 'v0.1.10' }}
```

Comment:
There's a `version` script at the root that used to update this version during `npm version`: https://github.com/pkgjs/action/blob/main/package.json#L9 - not sure what's the best approach for it now.